App crash in emulator

**Error Log**
```
09-18 15:34:55.247 8917-8917/ubilabs.net.trufiBeta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ubilabs.net.trufiBeta, PID: 8917
    java.lang.RuntimeException: Unable to start activity ComponentInfo{ubilabs.net.trufiBeta/ubilabs.net.trufiBeta.MainActivity}: java.lang.IllegalStateException: sensorManager.getDefaultSensor(sensorType) must not be null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
        at android.app.ActivityThread.access$800(ActivityThread.java:151)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5254)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
     Caused by: java.lang.IllegalStateException: sensorManager.getDefaultSensor(sensorType) must not be null
        at com.hemanthraj.fluttercompass.FlutterCompassPlugin.<init>(FlutterCompassPlugin.kt:31)
        at com.hemanthraj.fluttercompass.FlutterCompassPlugin.<init>(FlutterCompassPlugin.kt:11)
        at com.hemanthraj.fluttercompass.FlutterCompassPlugin$Companion.registerWith(FlutterCompassPlugin.kt:25)
        at com.hemanthraj.fluttercompass.FlutterCompassPlugin.registerWith(FlutterCompassPlugin.kt)
        at io.flutter.plugins.GeneratedPluginRegistrant.registerWith(GeneratedPluginRegistrant.java:19)
        at ubilabs.net.trufiBeta.MainActivity.onCreate(MainActivity.kt:11)
        at android.app.Activity.performCreate(Activity.java:5990)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2278)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387) 
        at android.app.ActivityThread.access$800(ActivityThread.java:151) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:135) 
        at android.app.ActivityThread.main(ActivityThread.java:5254) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:372) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698) 
```

--- a/android/src/main/kotlin/com/hemanthraj/fluttercompass/FlutterCompassPlugin.kt
+++ b/android/src/main/kotlin/com/hemanthraj/fluttercompass/FlutterCompassPlugin.kt
@@ -14,7 +14,7 @@ class FlutterCompassPlugin private constructor(context: Context, sensorType: Int
   private var mFilter = 1f
   private var sensorEventListener: SensorEventListener? = null
   private val sensorManager: SensorManager
-  private val sensor: Sensor
+  private var sensor: Sensor?
   private val orientation = FloatArray(3)
   private val rMat = FloatArray(9)
 

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -9,7 +9,7 @@ dependencies:
     sdk: flutter
 
 environment:
-    sdk: ">=1.19.0 <2.0.0"
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
 
 # For information on the generic Dart part of this file, see the
 # following page: https://www.dartlang.org/tools/pub/pubspec

